ScoutSuite/providers/gcp/resources/cloudsql/backups.py

```diff
@@ -0,0 +1,24 @@
+from ScoutSuite.providers.gcp.facade.gcp import GCPFacade
+from ScoutSuite.providers.base.configs.resources import Resources
+
+class Backups(Resources):
+    def __init__(self, gcp_facade: GCPFacade, project_id: str, instance_name: str):
+        self.gcp_facade = gcp_facade
+        self.project_id = project_id
+        self.instance_name = instance_name
+
+    async def fetch_all(self):
+        raw_backups = await self.gcp_facade.cloudsql.get_backups(self.project_id, self.instance_name)
+        for raw_backup in raw_backups:
+            if raw_backup['status'] == 'SUCCESSFUL':
```

Comment:
I guess it was like that before, but what about the failed backups?

Reply:
I guess they are considered not to be relevant, but @j4v would know more!